sdx_controller/handlers/connection_handler.py

```diff
@@ -286,7 +286,7 @@ def archive_connection(self, service_id) -> None:
             self.db_instance.add_key_value_pair_to_db(
                 MongoCollections.HISTORICAL_CONNECTIONS,
                 service_id,
-                [json.dumps({timestamp: json.loads(connection_request_str)})],
+                [json.dumps({timestamp: connection_request})],
```

Comment:
what about we also store Mongo Dict objects for HISTORICA_CONNECTIONS, as you did for CONNECTIONS? Instead of converting to string to then save to DB?

Reply:
Sounds good! I also updated this. Was planning to do the connections first, and do other collections in a follow up PR.